apps/backoffice-v2/src/pages/BusinessesAlerts/components/BusinessAlertsTable/utils/get-severity-from-risk-score.ts

```diff
@@ -0,0 +1,17 @@
+import { AlertSeverity, TAlertSeverity } from '@/domains/alerts/fetchers';
+
+export const getSeverityFromRiskScore = (riskScore: number): TAlertSeverity => {
+  if (riskScore <= 39) {
+    return AlertSeverity.LOW;
+  }
+
+  if (riskScore <= 69) {
+    return AlertSeverity.MEDIUM;
```

Comment:
Change instances of `MEDIUM` with `MODERATE`.
```suggestion
    return AlertSeverity.MODERATE;
```